### PR TITLE
Added support, if countdown lasts longer than 100 days

### PR DESCRIPTION
**Pull request by basert**
Now it's possible to last the countdown for more than 100 days due to 3 digits for the days

### CODE REVIEW

**Comment by martinaglv** [comment]
Looks good! The only suggestion that I have is to maybe show the triple digits only if the days left are more than 99.

**Comment by basert** [comment]
I've just needed the 100 digit support for myself, so it's a quick fix which i wanted to share :D But i will check :P

**Comment by basert** [comment]
Fixed!

File: countdown/jquery.countdown.js [hunk 1]
```diff
@@ -6,69 +6,75 @@
...
-			switchDigit(positions.eq(minor),Math.floor(value/10)%10);
-			switchDigit(positions.eq(major),value%10);
+			switchDigit(positions.eq(minor),Math.floor(value/10)%10, minor);
+			switchDigit(positions.eq(major),value%10, major);
 		}
-		
+
+        function updateTrio(minor, middle, major, value) {
+            switchDigit(positions.eq(minor),Math.floor(value/100)%10, minor);
+            switchDigit(positions.eq(middle),Math.floor(value/10)%10), middle;
```
Comment (zuma89) [inline countdown/jquery.countdown.js]: missed a `)` at this line.

**Comment by basert** [comment]
uhhh, :D just seen this :dancer:

**Pull request closed without merging**